include/inexor/vulkan-renderer/world/cube.hpp

```diff
@@ -44,6 +44,20 @@ class Cube : public std::enable_shared_from_this<Cube> {
     /// Cube Type.
     enum class Type { EMPTY = 0b00U, SOLID = 0b01U, NORMAL = 0b10U, OCTANT = 0b11U };
 
+    /// IDs of the children and edges which will be swapped to receive the rotation.
+    /// To achieve a 90 degree rotation the 0th index have to be swapped with the 1st and the 1st with the 2nd, etc.
+    struct RotationAxis {
+        using ChildType = std::array<std::array<std::size_t, 4>, 2>;
+        using EdgeType = std::array<std::array<std::size_t, 4>, 3>;
+        using Type = std::pair<ChildType, EdgeType>;
+        /// IDs of the children / edges which will be swapped to receive the rotation around X axis.
+        static constexpr Type X{{{{0, 1, 3, 2}, {4, 5, 7, 6}}}, {{{2, 4, 11, 1}, {5, 7, 8, 10}, {0, 9, 6, 3}}}};
+        /// IDs of the children / edges which will be swapped to receive the rotation around Y axis.
+        static constexpr Type Y{{{{0, 4, 5, 1}, {2, 6, 7, 3}}}, {{{{0, 5, 9, 2}}, {{3, 8, 6, 11}}, {{1, 10, 7, 4}}}}};
```

Comment:
Why the four brackets here?:
```cpp
static constexpr Type Y{{{{0, 4, 5, 1}, {2, 6, 7, 3}}}, {{{{0, 5, 9, 2}}, {{3, 8, 6, 11}}, {{1, 10, 7, 4}}}}};
                                                        ^^^^
```
For X and Z, three brackets seem enough.

Reply:
uuupppps your right.